### ISSUE TITLE
RawGit is shutting down. Please revoke any url points to it.

### ISSUE THREAD

**Issue by indrakaw** [Szero/volaupload.sh#7]
```sh
bash <(curl -sLo- https://raw.githubusercontent.com/szero/volascripts.sh/master/install.sh)
```
Is work fine. I have to replace all string on `install.sh`, from `https://rawgit.com/` to `https://raw.githubusercontent.com/`, thought.

Got better solutions?

**Comment by indrakaw** [comment]
Rename it to somewhat like this  `https://github.com/szero/volascripts.sh/raw/master/install.sh` would works.

**Comment by szero** [comment]
Hey thanks, I wasn't aware of that, fixed here c106777
